### ISSUE TITLE
Hook up point geocoding

### ISSUE THREAD

**Issue by cwlawlis802** [nationalparkservice/npmap.js#375]
Separating out into its own issue from https://github.com/nationalparkservice/places-data/issues/543#issuecomment-185963114.

> The geocoder should look in `parks_poly` for a geometry first, if null look in `parks_point` for `geom_point` on the same unit code.

**Comment by nateirwin** [comment]
This is already hooked up: https://github.com/nationalparkservice/npmap.js/blob/master/src/control/geocoder.js#L203-L212. Are you seeing issues with this, @cwlawlis802?

**Comment by cwlawlis802** [comment]
It doesn't seem to be working for units with only a label point. ALEU, for example. It only has a point geometry in the `parks_label` table, so maybe it needs a point in the `parks_point` table? Units like these do not have area geometries.

Where is the geocoder looking for the point geometry?

**Comment by nateirwin** [comment]
This is still working against parks_v1. Hitting a label point is a separate issue altogether, and the structure of _v2 makes it difficult to do operations like this. Let's chat about this more and figure out the best path forward.

**Comment by cwlawlis802** [comment]
Sounds good, I'll find a time for us to get together with @jimmyrocks and @mtaylorlong.

**Comment by cwlawlis802** [comment]
Issue for new schema implementation is here https://github.com/nationalparkservice/places-data/issues/732.

Once this is complete you are good to hook this up @nateirwin. Thanks!

**Comment by nateirwin** [comment]
Done!
